.github/workflows/release.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 The Protobom Authors
+# SPDX-License-Identifier: Apache-2.0
+
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write # needed to write releases 
+
+    steps:
+      - name: Set tag name
+        shell: bash
+        run: |
+          echo "TAG=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v3
+        with:
+          go-version: '1.22'
+          check-latest: true
+          cache: true
```

Comment:
Let's disable caching, we started doing that for minder and other projects because of some recent exploit leveraging that

Reply:
OK, done!